abi/src/lib.rs

```diff
@@ -163,6 +167,7 @@ pub enum Value {
     Int32(i32),
     Int64(i64),
     Hash([u64; 4]),
+    Hash8([u64; 8]),
```

Comment:
It's a kind of constant-size array, faster than using an array(closer to Vec in Rust) because we know the size at compile time. We could also replace them with u128 and u256.

Reply:
Can we not just have a constant sized array type?

Reply:
We could, should we do that in a separate issue?

Reply:
Ok, let's move ahead with this for now then, and I'll create another for the array type.